### PR TITLE
Add misc files for publishing

### PR DESCRIPTION
**Pull request by markwolff**
- Add main entry point in rollup and package.json
- Add typedoc comments
- Add readme to all sub-repos
- Rename AISKU readme-v2 to README.md
- Rename applicationinsights-sdk-js to applicationinsights-web

### CODE REVIEW

File: AISKU/package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
     "name": "@microsoft/applicationinsights-web",
-    "version": "1.0.0-beta.2",
+    "version": "1.0.0-beta",
     "description": "Microsoft Application Insights Javascript SDK API 1.0 beta",
-    "main": "dist-esm/applicationinsights-sdk-js.js",
-    "types": "types/applicationinsights-sdk-js.d.ts",
+    "main": "dist/applicationinsights-web.min.js",
+    "module": "dist-esm/applicationinsights-web.js",
+    "types": "types/applicationinsights-web.d.ts",
```
Comment (jpiyali) [inline AISKU/package.json]: totally unrelated to pr: did the api extractor plugin work to roll up interfaces to single file?

File: AISKU/src/Initialization.ts [hunk 2]
```diff
@@ -19,16 +24,20 @@ export interface IApplicationInsights extends IAppInsights, IDependenciesPlugin,
     appInsights: ApplicationInsights;
 };
 
+/**
+ * Application Insights API
+ * @class Initialization
+ * @implements {IApplicationInsights}
+ */
 export class Initialization implements IApplicationInsights {
```
Comment (jpiyali) [inline AISKU/src/Initialization.ts]: One thing we are missing here is making channels available or just the flush api. its a common requirement.

Comment (jpiyali) [reply AISKU/src/Initialization.ts]: Internally, it will call getchannelcontrols().foreach(c => c.flush)

Comment (markwolff) [reply AISKU/src/Initialization.ts]: got it. will add for next beta version

File: AISKU/package.json (hunk 1 above)
Comment (markwolff) [reply AISKU/package.json]: It left all of the external types as external instead of trying to resolve them

**Pull request merged**